zarr/tests/test_storage_v3.py

```diff
@@ -235,6 +254,67 @@ def test_rename_nonexisting(self):
             with pytest.raises(NotImplementedError):
                 store.rename('a', 'b')
 
+    def test_get_partial_values(self):
+        store = self.create_store()
+        store.supports_efficient_get_partial_values()
```

Comment:
```suggestion
        assert store.supports_efficient_get_partial_values()
```

Reply:
This should not be asserted here, since not all stores have the efficient access. I just added it to ensure that the call works. I added an `in [True, False]` check instead.